dubbo-rpc/dubbo-rpc-triple/src/main/java/org/apache/dubbo/rpc/protocol/tri/ReflectionPackableMethod.java

```diff
@@ -390,12 +393,27 @@ public byte[] pack(Object obj) throws IOException {
             }
             final TripleCustomerProtocolWapper.TripleRequestWrapper.Builder builder = TripleCustomerProtocolWapper.TripleRequestWrapper.Builder.newBuilder();
             builder.setSerializeType(serialize);
-            for (Object argument : arguments) {
-                builder.addArgTypes(argument.getClass().getName());
+
+            for (int i = 0 ; i < arguments.length; i++) {
+                Object argument = arguments[i];
+                String paramTypeName;
+                Class<?> paramTypeClass;
+
+                if (argument == null) {
+                    paramTypeName = actualRequestTypes[i].getName();
+                    paramTypeClass = actualRequestTypes[i];
+                }else{
+                    paramTypeName = arguments[i].getClass().getName();
+                    paramTypeClass = arguments[i].getClass();
+                }
+
```

Comment:
The parameter type has been passed in, so there is no need to use arguments[i].getClass()